contracts/LifToken.sol

```diff
@@ -12,4 +12,54 @@ contract LifToken is MintableToken, Pausable {
 
   // Token decimals
   uint public constant DECIMALS = 18;
+
+  // Edit of the ERC20 token events to support data argument
```

Comment:
I'd do this:

"transferData is an addition to ERC20 token methods. It allows to transfer value and data on each transaction"